src/mbgl/map/map.cpp

```diff
@@ -99,6 +101,7 @@ Map::Impl::Impl(View& view_,
       contextMode(contextMode_),
       pixelRatio(view.getPixelRatio()),
       asyncUpdate([this] { update(); }),
+      workerThreadPool(4),
```

Comment:
Agreed. I think we can target that for a followup that also makes the thread pool global, instead of per-`Map`.